.golangci.yml

```diff
@@ -162,5 +162,6 @@ run:
 
 issues:
   exclude:
-    # Naming of `error` variables as `err` is standard practise for Go
-    - 'shadow: declaration of "err" shadows declaration at'
+  # Naming of `error` variables as `err` is standard practise for Go
+  - 'shadow: declaration of "err" shadows declaration at'
+  - '`cancelled` is a misspelling of `canceled`'
```

Comment:
THANK YOU! I also see `favour` as an ignored misspelled word. Should we add cancelled/canceled in that section instead of here?